README.md

```diff
@@ -21,11 +21,11 @@ These methods are "panic safe", meaning that if a call to external code (e.g., a
 * Items in `0..len()` will only be items originally in the `Vec` or items being added to the `Vec`. It will never include uninitialized memory, duplicated items or dropped items.
 * Items originally (but no longer) in the `Vec` or being added to (but not yet in) the `Vec` may be leaked - any method that may leak items like this will have a note to specify its behavior.
 
-The exact behavior of each method is specified in its documentations.
+The exact behavior of each method is specified in its documentation.
```

Comment:
Unsure whether this was actually intentional, as I believe it can technically be a correct plural form, but think such usage is exceedingly rare, and also doesn't seem there would be multiple per method anyway

Reply:
Yep, it's a typo: thanks for fixing this!